astropy/io/ascii/cds.py

```diff
@@ -42,12 +42,17 @@ def get_type_map_key(self, col):
 
 
     def get_cols(self, lines):
-        """Initialize the header Column objects from the table ``lines`` for a CDS
+        """
+        Initialize the header Column objects from the table ``lines`` for a CDS
         header.
 
-        :param lines: list of table lines
-        :returns: list of table Columns
+        Parameters
+        ----------
+        lines : list
+            List of table lines
+
```

Comment:
@adrn - yes, you are right.   I was just being misled by the original (incorrect) docs.